### PR TITLE
[MI-3288] Add content-type header in different events

### PR DESCRIPTION
**Pull request by ayusht2810**
#### Summary
Add content-type header in different events

#### Ticket Link
https://github.com/mattermost/mattermost-plugin-msteams-sync/issues/213

### CODE REVIEW

**Review by esarafianou — APPROVED** [review]

**Comment by manojmalik20** [comment]
Are we sure that setting the conent type in response will save us from attacks? @esarafianou @jespino

**Review by jespino — CHANGES_REQUESTED** [review]
Actually the real content type there is `plain/text`, not `application/json`

**Review by jespino — APPROVED** [review]

**Comment by ayusht2810** [comment]
@jespino updated the content-type. Please re-review

**Comment by jespino** [comment]
@ayusht2810 looks like the CI is failing without a reason, can you take a look, maybe we need to merge the main branch or something like that.

File: server/store/store_test.go [hunk 1]
```diff
@@ -38,28 +37,30 @@ func setupTestStore(api *plugintest.API, driverName string) (*SQLStore, *plugint
...
 				},
 				Started: true,
 			})
 
-		hostPort, _ := postgres.MappedPort(context.Background(), postgresPort)
-		conn, _ := sqlx.Connect("postgres", fmt.Sprintf("postgres://user:pass@localhost:%s?sslmode=disable", hostPort.Port()))
+		time.Sleep(5 * time.Second)
+		host, _ := postgres.Host(context)
+		hostPort, _ := postgres.MappedPort(context, "5432/tcp")
+		conn, _ := sqlx.Connect("postgres", fmt.Sprintf("postgres://user:pass@%s:%d?sslmode=disable", host, hostPort.Int()))
```
Comment (manojmalik20) [inline server/store/store_test.go:61]:
```suggestion
		conn, _ := sqlx.Connect(driverName, fmt.Sprintf("postgres://user:pass@%s:%d?sslmode=disable", host, hostPort.Int()))
```

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**